### PR TITLE
Issue 7 updated events

### PR DESCRIPTION
**Pull request by jamiehewitt15**
Closes: #7

Changes proposed in this PR:

- Updated documentation
- Checking for the auth service in the event that is received.
- Creating separate auth modules

### CODE REVIEW

File: README.md
Comment (dungeonhao) [inline README.md:57]: Please add `"authService": "keycloak"` in the JSON snippets.

Comment (dungeonhao) [reply README.md:57]: Other than this, it should be fine

Comment (jamiehewitt15) [reply README.md:59]: Ok, thanks for the feedback. I've updated the readme

**Comment by jamiehewitt15** [comment]
@dungeonhao I'm still waiting for tour approval on this

**Review by dungeonhao — APPROVED** [review]

**Pull request merged**
